lib/validation.ts

```diff
@@ -187,7 +189,7 @@ export function fileExists(_path: string): boolean {
   return true;
 }
 
-export function checkAndConvertToJson(_path: string): object | null {
+export function checkAndConvertToJson(_path: string): Schema | null {
```

Comment:
This generic util doesn't know which type of object it's parsing the JSON for. It could be a schema, a hubdb table, an object, or something else. This seems like a good place to leverage TS generic function typing.